### PR TITLE
Add option to toggle the tooltip in memberCount.

### PR DESCRIPTION
**Pull request by cheesesamwich**
https://github.com/Vencord/plugin-requests/issues/128 :3

### CODE REVIEW

**Comment by vrad-exe** [comment]
Thanks for adding this! You should probably do the opposite too, and make it so you can turn off the display in the member list and have it only display in the tooltip. I wouldn't personally use that option but it makes sense if you're adding a settings menu.

**Comment by cheesesamwich** [comment]
good idea, ill do it soon.

**Comment by Nuckyz** [comment]
Use an Enum instead of hardcoding numebrs

**Comment by Sqaaakoi** [comment]
> Use an Enum instead of hardcoding numebrs

an array of enums mapped to strings and calling settings.store.views.includes(enum) is a more specific way of doing it
would also be more extensible in the future

**Comment by cheesesamwich** [comment]
> Use an Enum instead of hardcoding numebrs

done :P

File: src/plugins/memberCount/index.tsx [hunk 1]
```diff
@@ -32,6 +33,25 @@ export const ChannelMemberStore = findStoreLazy("ChannelMemberStore") as FluxSto
     getProps(guildId: string, channelId: string): { groups: { count: number; id: string; }[]; };
 };
 
+enum viewTypes {
```
Comment (Vendicated) [inline src/plugins/memberCount/index.tsx:36]:
```suggestion
const enum ViewTypes {
```

File: src/plugins/memberCount/index.tsx [hunk 2]
```diff
@@ -32,6 +33,25 @@ export const ChannelMemberStore = findStoreLazy("ChannelMemberStore") as FluxSto
...
+    MemberList,
+    Tooltip,
+    Both
+}
+
+const settings = definePluginSettings({
+    views: {
+        type: OptionType.SELECT,
+        description: "Where the member count should be displayed.",
```
Comment (Vendicated) [inline src/plugins/memberCount/index.tsx:45]:
```suggestion
        description: "Where the member count should be displayed",
```

**Comment by Vendicated** [comment]
i don't like the multiple choice because it is not extendable... what if we add it in a third place? we'd have to change how it works and migrate old settings. it would be better to just use multiple boolean options

**Comment by Sqaaakoi** [comment]
> i don't like the multiple choice because it is not extendable... what if we add it in a third place? we'd have to change how it works and migrate old settings. it would be better to just use multiple boolean options

I think Vencord needs a dedicated "select one or more" settings type for these kinds of things at this point

**Comment by vrad-exe** [comment]
It should probably just be 2 toggles, and just let the user turn all of them off if they really want to do that for some reason

**Comment by Sqaaakoi** [comment]
> just let the user turn all of them off if they really want to do that for some reason

you severely underestimate how dumb some users are, I think a safety check for this is a good idea

**Comment by Vendicated** [comment]
don't add pointless checks. it doesn't hurt to have it on with no features enabled

**Pull request merged**